src/chi/components/accordion/accordion.scss

```diff
@@ -0,0 +1,54 @@
+@import '_variables';
+@import '_mixins';
+
+.chi-accordion {
+  background-color: set-color(white);
+  border-bottom: 1px solid set-color(grey,20);
+
+  &__header {
+    display: flex;
+    padding: 1rem;
+
+    .chi-accordion__title {
+      font-size: $text-larger;
+      font-weight: $font-weight-semibold;
```

Comment:
Since Accordion is a functional component that is supposed to expand and collapse by clicking the Accordion title, let's add `cursor: pointer;` to it